src/main/java/org/opensearch/geospatial/ip2geo/common/Ip2GeoSettings.java

```diff
@@ -52,16 +52,11 @@ public class Ip2GeoSettings {
     );
 
     /**
-     * Multi search max concurrent searches
-     *
-     * Multi search is used only when a field contains a list of ip addresses.
-     *
-     * When the value is 0, it will use default value which will be decided
-     * based on node count and search thread pool size.
+     * Max size for geo data cache
      */
-    public static final Setting<Integer> MAX_CONCURRENT_SEARCHES = Setting.intSetting(
-        "plugins.geospatial.ip2geo.processor.max_concurrent_searches",
-        0,
+    public static final Setting<Long> CACHE_SIZE = Setting.longSetting(
+        "plugins.geospatial.ip2geo.processor.cache_size",
+        1000,
```

Comment:
why not 10k?

Reply:
This cache is created when Geospatial plugin is installed regardless a user uses Ip2Geo processor or not. So, I think it could be better to start with small number and let user to decide to increase the cache size later. Also, the performance test result with 1,000 wasn't really bad compared to 10,000.